src/pudl/metadata/resources/eia.py

```diff
@@ -1045,6 +1045,56 @@
         "sources": ["eia860", "eia923"],
         "etl_group": "outputs",
     },
+    # "core_eia__codes_cooling_tower_types": {
+    #     "description": "A coding table for cooling tower types.",
+    #     "schema": {
+    #         "fields": ["code", "label", "description"],
+    #         "primary_key": ["code"],
+    #         "foreign_key_rules": {
+    #             "fields": [
+    #                 ["tower_type_1", "tower_type_2", "tower_type_3", "tower_type_4"]
+    #             ]
```

Comment:
This is all commented out because it is somehow breaking the dagster launch. Need to dig more into why. If we wanted to leave all the encoding stuff out of this PR we could because it's more for the harvesting step anyways. I didn't realize how interconnected it all was. My initial thought was just to add the `CODE_METADATA` but the builds would fail without the coding tables depicted here (see [comment](https://github.com/catalyst-cooperative/pudl/pull/3405/files#r1497026657)). And then these cause dagster to fail. So need to figure out the chain of events.